src/Network/Wai/Log/Options.hs

```diff
@@ -93,17 +93,15 @@ defaultLogRequest uuid req =
 --
 -- Time is in seconds as that is how 'NominalDiffTime' is treated by default
 defaultLogResponse :: UUID -> Request -> Response -> Value -> ResponseTime -> [Pair]
-defaultLogResponse uuid req resp responseBody time =
-    [ "request_uuid"  .= uuid
-    , "method"        .= ts (requestMethod req)
-    , "url"           .= ts (rawPathInfo req)
-    , "response_body" .= responseBody
-    , "status" .= object [ "code"    .= statusCode (responseStatus resp)
-                         , "message" .= ts (statusMessage (responseStatus resp))
-                         ]
-    , "time"   .= object [ "full"    .= full time
-                         , "process" .= processing time
-                         ]
+defaultLogResponse uuid req resp responseInfo time =
```

Comment:
_Idea:_ as an alternative add a new `defaultLogResponseFlat` or something along those lines?
Then this could technically be a minor release: https://pvp.haskell.org/#decision-tree

Reply:
Agree with Jonathan that having it as an alternative default would save us from reacting to it as a breaking change.

Reply:
Is that a problem?

Reply:
We use it in the eID service, and we'll need to adjust a few graphs on the dashboards. Not a big problem, though. How does having these fields nested result in type conflicts?